Try to fix https://bugzilla.redhat.com/show_bug.cgi?id=452359; RH/Fedora
initscripts do not support killing of whole processgroups. Hence, disable
feature which posts negative pgrp into pidfile and use plain pid instead
of.

diff -up clamav-0.93.3/clamav-milter/clamav-milter.c.~1~ clamav-0.93.3/clamav-milter/clamav-milter.c
--- clamav-0.93.3/clamav-milter/clamav-milter.c.~1~	2008-07-07 20:02:10.000000000 +0200
+++ clamav-0.93.3/clamav-milter/clamav-milter.c	2008-07-13 20:15:12.000000000 +0200
@@ -1985,7 +1985,7 @@ main(int argc, char **argv)
 			logg(_("!Can't save PID in file %s\n"), pidfile);
 			return EX_CONFIG;
 		}
-#ifdef	C_LINUX
+#if defined(C_LINUX) && 0
 		/* Ensure that all threads are kill()ed */
 		fprintf(fd, "-%d\n", (int)getpgrp());
 #else
